.changes/next-release/Feature-546b77ba-20be-4573-811d-6a6d86b47f00.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Added 'aws.aboutToolkit' command to show AWS Toolkit versioning details that are useful to include with bug reports."
```

Comment:
Oops, I meant the user facing command text.  Users don't know what "aws.aboutToolkit" is or how to access it.

Reply:
That makes sense, I've changed it and mentioned the menu option in the latest commit.